### DISCUSSION TITLE
best practice for later transision away from php-backend

### DISCUSSION THREAD

**Discussion by s-light** [feathersjs/feathers#2857] [Q&A] [answered]
Hello Folks,

do you have any tips ideas on this scenario:
there is a web *application* currently using all-logic in php with static front-end (simple forms).

i like to slowly convert this to use vuejs as framework and feathers for the communication to the db/backend.
so in my first steps i plan to rewrite some very small parts of the frontend as pwa in vue - 
still with the php backend to the db.
are there any good-practice ideas so i can easyly later switch to a full feathersjs backend?
so for example have some client-side *poly-fills* (to convert from the feathersjs to the php calls?)

or does this make no sens at all??

looking forward to your ideas and thoughts :-)

sunny greetings
stefan

**Comment by noor-tg** [comment, marked as answer]
it is better to start from backend .
If you use template engine now. Just convert from as example blade to ejs.

**Reply by s-light** [reply]
thanks for your feedback.
goes in the direction i thought....

its best to not make it as simple as possible ;-)
